app/jobs/subscription_placement_job.rb

```diff
@@ -29,10 +29,16 @@ def proxy_orders
 
   def place_order_for(proxy_order)
     JobLogger.logger.info("Placing Order for Proxy Order #{proxy_order.id}")
-    proxy_order.initialise_order!
+    initialise_order(proxy_order)
     place_order(proxy_order.order)
   end
 
+  def initialise_order(proxy_order)
+    proxy_order.initialise_order!
+  rescue StandardError => e
+    Bugsnag.notify(e, subscription: proxy_order.subscription, proxy_order: proxy_order)
```

Comment:
There's no actual need for the call to notify. Workers already notify unhandled exceptions in [config/initializers/delayed_job.rb](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/initializers/delayed_job.rb). If what we want is the extra diagnosis details from the subs. I'd rather go with https://docs.bugsnag.com/platforms/ruby/rails/#attaching-custom-diagnostics in that initializer. We already do it in few places.

Reply:
As mentioned above, this is already the solution for #4983 (continue processing the rest of the orders), it's a simple catch all and log that enables the job to continue doing what's doing. I think this is way easier and equally effective than creating a job to process each proxy_order.